clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/avoid-do-while.rst

```diff
@@ -26,7 +26,8 @@ Options
 
 .. option:: IgnoreMacros
 
```

Comment:
Personally I don't see much benefit in this change, it makes the documentation harder to read as one has to go back and forth. I don't see the trouble with this duplication, since it's not really a problem if the sentences don't match exactly with each other - it does not add maintenance cost.

Sometimes DRY can cause more trouble that it's worth - a bit of duplication makes sense in benefit of usability and readability.